### DISCUSSION TITLE
How far did I cycle in total?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#576] [Galaxy Trivia] [answered]
Yesterday I went for a short bicycle ride around the local lakes. As the weather was very hot, I rode in various stages.

In the first stage I rode half of the overall distance.
Stage two saw half of the remaining distance plus 35 meters covered.
Stage three covered three-quarters of the remaining distance.
Stage four completed half of the remaining distance plus 75 meters.
Stage five completed the journey with a final burst of 150 meters.

**Comment by peterbaccaro** [comment, marked as answer]
**7340 metres**

Working backwards...

1) Stage 5 says 150 metres left.

2) Stage 4 says half of the remaining distance plus 75 meters.
   Distance at start of Stage 4 must have been 450 metres.

```
Dist4   = (Dist4/2) + 75 + 150 
2*Dist4 = Dist4 + 150 + 300
2*Dist4 = Dist4 + 450
Dist4   = 450 metres
```

3) Stage3 says that 450 metres was one quarter of remaining distance.
   Distance at start of Stage3 would have been 1800 metres.

```
Dist3 = 450 * 4
Dist3 = 1800 metres
```

4) Stage2 says that half of the remaining distance plus 35 meters covered.
   Distance at start of Stage2 would have been 3670 metres.

```
Dist2   = (Dist2/2) + 35 + 1800
2*Dist2 = Dist2 + 70 + 3600
2*Dist2 = Dist2 + 3670
Dist2   = 3670 metres
```

5) Stage1 tell us that this stage is half the distance.
   So full distance was 7340 metres.

```
Dist1 = 3670 * 2
Dist1 = 7340 metres
```

**Reply by ParanoidUser** [reply]
Good math!
